src/622/modules/genius/index.js

```diff
@@ -0,0 +1,35 @@
+const rules = {
+  vowel: {
+    err0: { blue: 'red', green: 'yellow', red: 'blue', yellow: 'green', },
+    err1: { blue: 'green', green: 'blue', red: 'yellow', yellow: 'red', },
+    err2: { blue: 'red', green: 'yellow', red: 'green', yellow: 'blue', },
+  },
+  "not-vowel": {
+    err0: { blue: 'yellow', green: 'green', red: 'blue', yellow: 'red', },
+    err1: { blue: 'blue', green: 'yellow', red: 'red', yellow: 'green', },
+    err2: { blue: 'green', green: 'blue', red: 'yellow', yellow: 'red', },
+  }
+}
+
+/**
+ * Função que resolve o módulo genius
+ * @param {Object} config - Objeto de configuração da bomba
+ * @param {boolean|null} config.serial-vowel - Indica se existe uma vogal no número de série
+ * @param {number} config.errors - Quantidade de erros na bomba
+ * @param {...colors} colors - Lista das cores na ordem que acendem
```

Comment:
arrumar documentação
```suggestion
 * @param {...string} colors - Lista das cores na ordem que acendem
```